docs/develop/sdk/1-getting-started-guide/00-getting-started.md

```diff
@@ -0,0 +1,179 @@
+---
+id: getting-started
+title: Kilt Getting Started Guide
+---
+import CodeBlock from '@theme/CodeBlock';
+import Example1 from '!!raw-loader!@site/code-examples/core_features/getting_started/1_getting_started.ts';
+import Example2 from '!!raw-loader!@site/code-examples/core_features/getting_started/2_getting_started.ts';
+import Example3 from '!!raw-loader!@site/code-examples/core_features/getting_started/3_getting_started.ts';
+import Example4 from '!!raw-loader!@site/code-examples/core_features/getting_started/4_getting_started.ts';
+import Example5 from '!!raw-loader!@site/code-examples/core_features/getting_started/5_getting_started.ts';
+import Example6 from '!!raw-loader!@site/code-examples/core_features/getting_started/6_getting_started.ts';
+import Example7 from '!!raw-loader!@site/code-examples/core_features/getting_started/7_getting_started.ts';
+import Example8 from '!!raw-loader!@site/code-examples/core_features/getting_started/8_getting_started.ts';
+import Example9 from '!!raw-loader!@site/code-examples/core_features/getting_started/9_getting_started.ts';
+import Example10 from '!!raw-loader!@site/code-examples/core_features/getting_started/10_getting_started.ts';
+
+The following guide will give you a started point to begin with KILT.
+What will you be able to do after this guide:
+
+1. Import the **KILT SDK** into a project
+2. Connect to the **KILT blockchain**
+3. Query a **web3name** to get a **DID**
+4. Verify a **credential**, fetched from **DID service endpoints**
+
+After completing the getting started, you should have learn a little about KILT and a guide to learn by doing.
+The guide requires some experience with javascript and command-line tools.
+We will have recommended guides to other tutorials for further learnings.
+
+## Setup
+
+We will focus on creating a new project from stratch, which will require a little setup.
+First, we need to create a new project in a new directory. For this, we run `mkdir kilt-rocks && cd kilt-rocks`.
+
+From inside the `kilt-rocks` project directory, install the **SDK**, **node** and **node-fetch** with either of the following package managers:
+
+```bash npm2yarn
+npm install @kiltprotocol/sdk-js node node-fetch
+```
+
+After you have imported the SDK, you are now able to access the functionality of KILT.
+With all the required dependencies set, just create a new (empty) script file with `touch getting-started.js`.
+Inside the `package.json` add in the object `"type": "module"`.
+
+Lets first create an asynchronous function called `main` in order to excute the script.
+Underneath the first line add the following:
+
+<CodeBlock className="language-js">
+  {Example1}
+</CodeBlock>
+
+If the setup is correct you can excute the script by calling the name of the file using node.
+
+```bash npm2yarn
+npm node getting-started.js
```

Comment:
This shows as `# couldn't auto-convert command`.

Reply:
Which is strange since it works as intended.

Reply:
Just saying we probably want to change the command, as the rendered version somehow shows the error.

Reply:
I have a workaround. It isn't pretty but does the trick.

Reply:
Looks better 👍